pyfar/classes/audio.py

```diff
@@ -1497,87 +1512,82 @@ def _matrix_multiplication(a, b, axes, audio_type):
         a = np.expand_dims(a, 0) if a.ndim == 2 else a
         b = np.expand_dims(b, 1) if b.ndim == 2 else b
         # note: axes is implicitly copied
-        axes = [tuple([ax-1 if ax < 0 else ax for ax in t]) for t in axes]
+        axes = [tuple([ax - 1 if ax < 0 else ax for ax in t]) for t in axes]
     return np.matmul(a, b, axes=axes)
 
 
-def _match_fft_norm(fft_norm_1, fft_norm_2, division=False):
+def _match_fft_norm(fft_norm1, fft_norm2, operation, norm_override=None):
     """
-    Helper function to determine the fft_norm resulting from an
-    arithmetic operation of two audio objects.
-
-    For addition, subtraction and multiplication:
-    Either: one signal has fft_norm ``'none'`` , the results gets the other
-    norm.
-    Or: both have the same fft_norm, the results gets the same norm.
-    Other combinations raise an error.
-
-    For division:
-    Either: the denominator (fft_norm_2) is ``'none'``, the result gets the
-    fft_norm of the numerator (fft_norm_1).
-    Or: both have the same fft_norm, the results gets the fft_norm ``'none'``.
-    Other combinations raise an error.
```

Comment:
This actually gives more information than the new version, which might be helpfull for debugging. We should keep the old version IMO